src/components/DateRangePicker.js

```diff
@@ -416,18 +419,16 @@ const DateRangePicker = React.createClass({
       // Selected Date styles
       selectedDateWrapper: {
         alignItems: 'center',
-        cursor: 'pointer',
         display: 'flex',
-        justifyContent: 'space-between',
-        position: this.props.isRelative ? 'relative' : 'static'
+        height: 20,
+        justifyContent: 'space-between'
```

Comment:
position rule & cursor was not necessary because it is already on the component level rules.

Without height the component was using vertical padding to resize to the weird sizes.